### PR TITLE
all: add CheckerParam support

### PR DESCRIPTION
**Pull request by Quasilyte**
Updates #17

Signed-off-by: Iskander Sharipov <quasilyte@gmail.com>

### CODE REVIEW

File: linter/lintmain/internal/check/check.go [hunk 1]
```diff
@@ -246,6 +249,29 @@ func (l *linter) loadPlugin() error {
 }
 
 func (l *linter) parseArgs() error {
+	paramKey := func(info *lintpack.CheckerInfo, paramName string) string {
+		return "@" + info.Name + "." + paramName
```
Comment (cristaloleg) [inline linter/lintmain/internal/check/check.go]: I love this pattern with a local func in your code

File: linter/lintmain/internal/check/check.go [hunk 2]
```diff
@@ -246,6 +249,29 @@ func (l *linter) loadPlugin() error {
...
+	}
+
+	intParams := make(map[string]*int)
+	boolParams := make(map[string]*bool)
+	stringParams := make(map[string]*string)
+	for _, info := range l.infoList {
```
Comment (cristaloleg) [inline linter/lintmain/internal/check/check.go]: What about extracting this and following loops to a func?

Comment (Quasilyte) [reply linter/lintmain/internal/check/check.go]: Done.

Comment (cristaloleg) [reply linter/lintmain/internal/check/check.go]: Nice, LGTM

File: linter/lintmain/internal/check/check.go (hunk 1 above)
Comment (Quasilyte) [reply linter/lintmain/internal/check/check.go]: Can't be used when a function is needed in a more than 1 method. :)

Comment (cristaloleg) [reply linter/lintmain/internal/check/check.go]: well, yeah ;D

**Pull request merged**